modules/microsite/docs/scala/dropwizard/index.md

```diff
@@ -15,7 +15,7 @@ In addition to standard Dropwizard dependencies, you'll need:
 * `io.dropwizard:dropwizard-forms`
 * `com.datasift.dropwizard.scala:dropwizard-scala-core`
 * `com.fasterxml.jackson.datatype:jackson-datatype-jsr310`
-* `com.fasterxml.jackson.module:jackson-module-scala`
+* `com.fasterxml.jackson.module:jackson-module-scala_${scala_binary_version}`
```

Comment:
Should probably be [`scala.compat.version`](https://davidb.github.io/scala-maven-plugin/compile-mojo.html#scalaCompatVersion) as users that have to worry about this will likely be using scala-maven-plugin

Reply:
This is just for documentation purposes, but I guess it can't hurt.